docs/intro.rst

```diff
@@ -0,0 +1,111 @@
+Camus
+================
+
+Camus is a simple MapReduce job developed by LinkedIn to load data from Kafka into HDFS.
+It is capable of incrementally copying data form Kafka into HDFS such that
+every run of the MapReduce job picks up from where the previous run left off.
+At LinkedIn, Camus is used to load billions of messages per day from Kafka into HDFS.
+Confluent's version of Camus integrates with Confluent's schema registry which automatically
+validates the data integrity as it is loaded and ensure compatibility of data stored in HDFS
```

Comment:
"validates the data integrity" is a little confusing. Did you mean to say that Camus reliably loads data from Kafka into HDFS with no data loss?